modules/taboolaBidAdapter.md

```diff
@@ -0,0 +1,49 @@
+# Overview
+
+```
+Module Name: Taboola Adapter
+Module Type: Bidder Adapter
+Maintainer: prebid@taboola.com
+```
+
+# Description
+
+Module that connects to Taboola bidder to fetch bids.
+- Supports `display` format
+- Uses `OpenRTB` standard
+
+The Taboola Bidding adapter requires setup before beginning. Please contact us on prebid@taboola.com
+
+# Test Display Parameters
+``` javascript
+ var adUnits = [{
+  code: 'your-unit-container-id',
+  mediaTypes: {
+    banner: {
+      sizes: [[300, 250], [300,600]]
+    }
+  },
+  bids: [{
+    bidder: 'taboola',
+    params: {
+      tagId: 'Placement Name',
+      publisherId: 'your-publisher-id',
```

Comment:
Could you update with tagId and publisherId that return test bids from your system